dev/doc/release-process.md

```diff
@@ -0,0 +1,96 @@
+# Release process #
+
+## As soon as the previous version branched off master ##
+
+- [ ] Create a new issue to track the release process where you can copy-paste
+  the present checklist.
```

Comment:
This is only a proposal to have a good way to track the release process. I think it is a good idea but it was never experimented by @maximedenes and I (because this checklist wasn't ready yet).
@silene Tell me if you think you will use this way of doing. If not, there is no point in keeping this step in the doc.

Reply:
It will depend whether the process happens to be linear or not. If it is strictly linear, I presumably won't bother creating a new issue. If I end up having to track several items in parallel, then yours is a good idea and I will copy this document.